targets/targets.json

```diff
@@ -4381,6 +4381,23 @@
         ],
         "device_name": "STM32WL55JCIx"
     },
+    "MCU_STM32WLe5xC": {
+        "inherits": [
+            "MCU_STM32WL"
+        ],
+        "public": false,
+        "macros_add": [
+            "STM32WLE5xx"
+        ],
+        "extra_labels_add": [
+            "STM32WLE5xC"
+        ],
+        "device_name": "STM32WLE5JCIx",
```

Comment:
device_name can not be defined at this STM32WLe5xC level.
This information can be only added in the target level.

Reply:
done